README.md

```diff
@@ -46,66 +47,97 @@ The following packages are required for building OpenKilda controller:
 We do not recommend upgrading pip and install docker-compose using the methods described below, bypassing the packer managers. Instead, please read the documentation for installing the [pip](https://pip.pypa.io/en/stable/installation/#upgrading-pip) and the [docker-compose](https://docs.docker.com/compose/install/).
 
 
-#### Dependency installation on Ubuntu 18.04 
+#### Dependency installation on Ubuntu 18.04
+
 For running a virtual environment (that is a Docker instance with the Open vSwitch service) it is required to have Linux kernel 4.18+ for OVS meters support.
 The following commands will install necessary dependencies on Ubuntu 18.04:
-```shell
-sudo apt install maven make openjdk-8-jdk openvswitch-switch python3-pip linux-generic-hwe-18.04 tox rsync
-```
-```shell
-sudo pip3 install --upgrade pip
-sudo pip3 install docker-compose
-```
-#### Dependency installation on Ubuntu 20.04
-The following commands will install necessary dependencies on Ubuntu 20.04:
-```shell
-sudo apt install maven make openjdk-8-jdk openvswitch-switch python3-pip tox rsync
-```
-
-```shell
-sudo apt install maven make openjdk-8-jdk openvswitch-switch tox
-```
+  Add Python's PPA repository
+  ```shell
+  sudo add-apt-repository -y ppa:deadsnakes/ppa
+  ```
+  Install required packages:
+  ```shell
+  sudo apt update && sudo apt-get install -y \
+    maven \
+    openjdk-8-jdk \
+    python \
+    python3.8 \
+    python3-pip \
+    virtualenv \
+    make \
+    tox \
+    rsync \
+    openvswitch-switch \
+    linux-generic-hwe-18.04
+  ```
+    Install required components for Python 3.6:
```

Comment:
Some problem with the formatting here. The text falls into the code block. And it renders it as a inline code instead of a code block. It might be because of indentation.